armv7m7-imxrt117x/syspage.c

```diff
@@ -436,11 +436,12 @@ void syspage_addEntries(u32 start, u32 sz)
 
 /* Program's functions */
 
-int syspage_addProg(void *start, void *end, const char *imap, const char *dmap, const char *name)
+int syspage_addProg(void *start, void *end, const char *imap, const char *dmap, const char *name, u32 flags)
 {
 	u8 imapID, dmapID;
 	unsigned int pos, len;
 	u32 progID = syspage_common.progsCnt;
+	const u8 isExec = (flags & flagSyspageExec) != 0;
```

Comment:
```suggestion
	const u32 isExec = (flags & flagSyspageExec) != 0;
```
Shorter and faster.
The same for other targets.

Reply:
Corrected, thank You.